src/api-explorer/v4-0/InvoicePay.swagger2.json

```diff
@@ -0,0 +1,379 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "SAP Concur partners with external payment providers for processing invoice payments. These payment providers are listed on the App Center and can integrate with the Invoice product by using the Invoice Pay APIs. Payment providers can get a list of all the payments authorized to be processed by them, and send back status of those payments.",
+    "title": "Invoice Payment Service API Documentation"
+  },
```

Comment:
Should have a `version` key in the `info` object.

Reply:
Added the version key in the info object and updating the branch.